core/src/services/sftp/backend.rs

```diff
@@ -55,8 +55,8 @@ use crate::*;
 /// - [x] write
 /// - [x] create_dir
 /// - [x] delete
-/// - [ ] copy
-/// - [ ] rename
+/// - [ ] copy (partially, only supoort when remote server has copy-file extension)
```

Comment:
We don't need explain this: OpenDAL already implement all features, it's server's own duty.